CHANGELOG.md

```diff
@@ -3,6 +3,10 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [7.5.1] - 2024-10-21
+### Added
+- Set tag to disable S3 logs intelligent tiering lifecycle policy.
```

Comment:
`Added variable to add extra tags to s3 buckets`

Reply:
Done.